CONTRIBUTING.md

```diff
@@ -1,303 +1,61 @@
+### 🐣 New to Git:
 
-The main issues for bug reporting are as follows:
-- [go-ipfs/issues](https://github.com/ipfs/go-ipfs/issues) - Issues related to running the Go implementation of IPFS.
-- [js-ipfs/issues](https://github.com/ipfs/js-ipfs/issues) - Issues related to running the js implementation of IPFS.
-- [notes/issues](https://github.com/ipfs/notes/issues) - For general notes and ideas.
-- [specs/issues](https://github.com/ipfs/specs/issues) - For protocol discussions.
+If you are completely new to git then run through these tutorials and get a feel of how git works.
 
-The [go-ipfs](https://github.com/ipfs/go-ipfs) issues use a template that will guide you through the process of reporting a bug. We will be adding this kind of issue template to other repositories as bug reports become more common.
 
-For all other questions/discussions, please visit the [discussion forum](https://discuss.ipfs.io)
+[Hello World · GitHub Guides](https://guides.github.com/activities/hello-world/)
 
-## Protocol Design
+[Beginner’s Guide to Using Git and GitHub \| Codementor](https://www.codementor.io/git/tutorial/git-github-tutorial-for-beginners) 
 
-When considering protocol design proposals, we are looking for:
 
-- A description of the problem this design proposal solves
-- Discussion of the tradeoffs involved
-- Review of other existing solutions
-- Links to relevant literature (RFCs, papers, etc)
-- Discussion of the proposed solution
+### 📖 Docs and JavaScript Code Style
 
-Please note that protocol design is hard, and meticulous work. You may need to review existing literature and think through generalized use cases.
 
-## Implementation Design
+[JS Style Guide](https://github.com/standard/standard)
 
-When considering design proposals for implementations, we are looking for:
 
-- A description of the problem this design proposal solves
-- Discussion of the tradeoffs involved
-- Discussion of the proposed solution
 
-## Community Improvement
+### 👋 Need Extra Help:
 
-The IPFS community requires maintenance of various "public infrastructure" resources. These include documentation, github repositories, CI build bots, and more. There is also helping new users with questions, spreading the word about IPFS, and so on. Soon, we will be planning and running conferences. Please get in touch if you would like to help out.
+Working on your first open source issue can be intimidating. We get this, and the IPFS team is here to help. If you need assistance, you can [reach us at IRC](https://docs.ipfs.io/community/irc/). You can also ask questions during community office hours which are held [between 16:30-17pm](https://protocol.zoom.us/j/234238677). We also offer one-on-one help- if you are stuck on a bug then schedule a session with an experienced contributor who can help you. <br/><br/>
```

Comment:
"between 16:30-17pm" => "between 16:30-17" or "between 4:30-5pm", but not both 😉

Also, what time zone?

You should link to the community calendar here.